src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/CombineCompressCacheType.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * 2007-2017 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2017 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+namespace PrestaShopBundle\Form\Admin\AdvancedParameters\Performance;
+
+use PrestaShopBundle\Form\Admin\Type\CommonAbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * This form class is responsible to generate the "Combine Compress Cache" form in Performance page
+ */
+class CombineCompressCacheType extends CommonAbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('smart_cache_css', 'Symfony\Component\Form\Extension\Core\Type\ChoiceType', array(
+                'choices'  => array(
```

Comment:
```php
'choices'  => array(
      $this->translator->trans('Yes', [], 'Admin.Global') => 1,
      $this->translator->trans('No', [], 'Admin.Global') => 0,
),
```
hihi

Reply:
it will be translated using `choice_translation_domain` when needed (which is the case here, just didn't finished to add this property everywhere.

Reply:
just noticed our form template was deprecated (the migration may have started with Symfony 2.7 imo...), so I've updated the template according to the latest one from Symfony: thanks for your hint :)

Reply:
For Crowdin, it is indeed an issue, we may add a new rule in translation tool because we also use translation domain rule in twig template (as @eternoendless hinted to me) so I guess theses keys won't be re-imported on Crowdin too, am I right?